### PR TITLE
Issue#22 put

### PR DESCRIPTION
**Pull request by trentspi**

### CODE REVIEW

File: calendar/controllers/user-controller.js [hunk 1]
```diff
@@ -47,8 +47,20 @@ module.exports = exports = {
         });
     },
     put: (req, res) => {
+    // put updates the user information based on the req parameter
```
Comment (nrhubbar) [inline calendar/controllers/user-controller.js]: Where do you update the entry, right now this is currently the same as a GET.

Comment (nrhubbar) [inline calendar/controllers/user-controller.js]: http://mongoosejs.com/docs/2.7.x/docs/updating-documents.html

Comment (trentspi) [inline calendar/controllers/user-controller.js]: I will update the function, I have only fixed some syntax / naming errors at this point.

Comment (nrhubbar) [inline calendar/controllers/user-controller.js]: It's usual practice to not merge non-functioning code into master. Although your code may work, it does not give expected behavior. I won't approve it, but maybe @technetos @DannyPeck will.

Comment (trentspi) [inline calendar/controllers/user-controller.js]: I didn't mean to merge to master, my apologies. I will revise the PR and update the function.

File: calendar/controllers/user-controller.js [hunk 2]
```diff
@@ -47,8 +47,19 @@ module.exports = exports = {
...
+     User.findById(req.params.id, {$set:req.body}, function(err, user)){
```
Comment (DannyPeck) [inline calendar/controllers/user-controller.js]: I think it needs to be findbyIdAndUpdate not just findById.

File: calendar/controllers/user-controller.js [hunk 3]
```diff
@@ -47,8 +47,19 @@ module.exports = exports = {
         });
     },
     put: (req, res) => {
+      // put updates the user information based on the req parameter
+      User.findByIdAndUpdate(req.params.id, {$set:req.body}, {new: true})
+        .then((user, err) => {
+          // query the user, then update the users infsudo reormation based on the passed arguments
+          if (err) {
+            res.status(500).send("error with put request");
```
Comment (DannyPeck) [inline calendar/controllers/user-controller.js]: change this to res.status(500).json(err) or just send back the err's message.

File: calendar/package.json [hunk 4]
```diff
@@ -3,7 +3,7 @@
   "version": "0.0.0",
   "private": true,
   "scripts": {
-    "start": "node ./bin/www"
+    "start": "env DB_URI=localhost NODE_ENV=development node ./bin/www"
```
Comment (DannyPeck) [inline calendar/package.json]: remove this as we will make this change in a separate PR.

**Comment by trentspi** [comment]
done, requested changes completed.
